.github/workflows/release-phase-2.yml

```diff
@@ -18,7 +18,8 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
-    permissions: {}
+    permissions:
+      contents: write
```

Comment:
This seems out of scope.

Reply:
That's true. I've moved it to #573 instead now.